pkg/azuredisk/nodeserver.go

```diff
@@ -543,6 +559,25 @@ func (d *Driver) getDevicePathWithLUN(lunStr string) (string, error) {
 	return newDevicePath, err
 }
 
+func (d *Driver) getDevicePathWithMountPath(mountPath string) (string, error) {
+	// currently not supported for mac environment
+	if runtime.GOOS == "darwin" {
```

Comment:
if this is for unit test, we could skip this test in unit test, not in core code

Reply:
Will do